pages/index.js

```diff
@@ -216,32 +215,41 @@ export default props => (
         }}
       >
         <Text as="p" variant="eyebrow" color="yellow">
-          Mentorship program
+          Marathon program
         </Text>
         <Heading as="h2" variant="title">
-          1-on-1 coaching with a professional.
+          Make your dream a reality
```

Comment:
I want to be very concrete about exactly what we’re providing—I feel like this line is on so many hackathon, startup accelerator, etc websites